csrankings-g.csv

```diff
@@ -1010,3 +1011,4 @@ Günther R. Raidl,TU Wien,https://www.ac.tuwien.ac.at/people/raidl,NOSCHOLARPAGE
 Günther Ruhe,University of Calgary,http://ruhe.cpsc.ucalgary.ca,5nL_z0oAAAAJ
 Günther Schatter,Bauhaus University Weimar,https://www.uni-weimar.de/de/medien/professuren/medieninformatik/vernetzte-medien/personen/apl-prof-dr-guenther-schatter,NOSCHOLARPAGE
 Günther Specht,University of Innsbruck,https://dbis-informatik.uibk.ac.at/specht,Csj85-oAAAAJ
+

```

Comment:
Quite a few people have submitted PRs of late that have empty lines added to the bottom of their files. Please let me know what editor you used; I may need to add a pass to clear these empty lines. Thanks.